scripts/setup_automation_secure.py

```
@@ -0,0 +1,614 @@
+            cmd: Command as list of strings (avoids shell injection)
+            **kwargs: Additional subprocess.run arguments
+            
+        Returns:
+            CompletedProcess result
+        """
+        # Ensure cmd is a list of strings
+        if not isinstance(cmd, list):
+            raise ValueError("Command must be a list of strings")
+        
+        # Sanitize each argument
+        sanitized_cmd = [shlex.quote(str(arg)) if ' ' in str(arg) else str(arg) for arg in cmd]
```

Ruff (0.11.9)

Local variable `sanitized_cmd` is assigned to but never used

Remove assignment to unused variable `sanitized_cmd`

(F841)